model-optimizer/mo/graph/graph.py

```diff
@@ -1097,6 +1101,27 @@ def rename_nodes(nodes: List[tuple]):
     for node, name in nodes:
         rename_node(node, name)
 
+
+def get_attribute_between_nodes(node1: Node, node2: Node, attr_name: str):
```

Comment:
And lets change function name to `get_edge_attribute_between_nodes`. Similar to the function below.